### ISSUE TITLE
require-jsdoc ignores TypeScript interface members

### ISSUE THREAD

**Issue by eric-parsons** [gajus/eslint-plugin-jsdoc#647]
I see some work has been done recently to support TypeScript interfaces for the `require-jsdoc` rule. Thanks! I can get it to require documentation for the interface itself, but unfortunately not for any of its members.
## Expected behavior

Given this TypeScript interface:
```js
export interface Foo {
    bar: number;
    baz: string;
    quux(): void;
}
```

I would expect lint warnings for missing documentation for `bar`, `baz`, and `quux`, given the presence of "TSPropertySignature" and "TSMethodSignature" in `contexts` in my config (see below).

## Actual behavior

I only receive a lint warning for the missing docs on the `Foo` interface, but not for any of its properties or methods.

## ESLint Config

```js
"jsdoc/require-jsdoc": [
    "warn",
    {
        "publicOnly": true,
        "require": {
            "ArrowFunctionExpression": true,
            "ClassDeclaration": true,
            "ClassExpression": true,
            "FunctionDeclaration": true,
            "FunctionExpression": true,
            "MethodDefinition": true
        },
        "contexts": [
            "ArrowFunctionExpression",
            "FunctionDeclaration",
            "FunctionExpression",
            "MethodDefinition",
            "Property",
            "TSDeclareFunction",
            "TSEnumDeclaration",
            "TSInterfaceDeclaration",
            "TSMethodSignature",
            "TSPropertySignature",
            "TSTypeAliasDeclaration",
            "VariableDeclaration"
        ],
        "checkGetters": true,
        "enableFixer": false
    }
]
```

## ESLint sample

See interface in Actual behavior above.

## Environment

- Node version:  12.18.0
- ESLint version  7.10.0
- `eslint-plugin-jsdoc` version: 30.6.3

**Comment by eric-parsons** [comment]
Thanks for the quick fix! Seems to be working. Now I have a bunch of documentation to write :smile:.
